platforms/enterprise/enterprise/src/main/java/org/gradle/internal/enterprise/impl/DefaultGradleEnterprisePluginBuildState.java

```diff
@@ -28,7 +28,7 @@
 import org.gradle.internal.time.Clock;
 import org.gradle.launcher.daemon.server.scaninfo.DaemonScanInfo;
 
-@ServiceScope(Scopes.BuildTree.class)
+@ServiceScope(Scopes.Gradle.class)
```

Comment:
🤔 I suppose moving this to `BuildTree` scope is difficult?

Reply:
The goal of this PR was not to introduce any semantic changes, but rather make the annotations correspond to the actual scopes the services are registered. The semantic changes (actually changing the service registry owning the service) should be done separately and by the team owning the service.

More specifically, for this service, there were [changes recently](https://github.com/gradle/gradle/pull/26082) where the scopes of related services were reviewed and adjusted. This service in particular [has been moved around](https://github.com/gradle/gradle/commit/5a415fcf50a678f9c23c7ebbb6b07dc7ed492d35), but then the changes got reverted before merging the PR. So, I would have assumed the correct scope for this service is `Gradle`.

If you think the `Gradle` scope is actually incorrect, I suggest making a separate change either before or after this PR. Making it before would be better, because then we can avoid polluting git history of this particular line.

Reply:
👍 Makes sense. I'll follow up on this.